Hardcode our preferred path for perl library directory.
Not intended for upstream.

--- openscap-1.3.7/swig/perl/CMakeLists.txt
+++ openscap-1.3.7/swig/perl/CMakeLists.txt
@@ -20,7 +20,7 @@ install(PROGRAMS ${CMAKE_CURRENT_BINARY_
                 DESTINATION ${CMAKE_INSTALL_DATADIR}/perl5/vendor_perl)
 else()
         install(TARGETS ${SWIG_MODULE_openscap_pm_REAL_NAME}
-               DESTINATION ${PERL_VENDORLIB})
+		DESTINATION ${PERL_ARCHLIB})
         install(PROGRAMS ${CMAKE_CURRENT_BINARY_DIR}/openscap_pm.pm
-               DESTINATION ${PERL_VENDORARCH})
+		DESTINATION ${PERL_VENDORLIB})
 endif()
